plat/common/arm/cpu_native.c

```diff
@@ -37,6 +37,8 @@
 #include <arm/psci.h>
```

Comment:
It should be supported on ARM32 now, right?

Reply:
Yes, I missed removing those guards, my bad.